letsencrypt/main.py

```diff
@@ -0,0 +1,716 @@
+"""Let's Encrypt main entry point."""
+from __future__ import print_function
+import atexit
+import functools
+import os
+import sys
+import zope.component
+
+import letsencrypt
+
+from letsencrypt import account
+from letsencrypt import client
+from letsencrypt import cli
+from letsencrypt import crypto_util
+from letsencrypt import colored_logging
+from letsencrypt import configuration
+from letsencrypt import constants
+from letsencrypt import errors
+from letsencrypt import interfaces
+from letsencrypt import le_util
+from letsencrypt import log
+from letsencrypt import reporter
+from letsencrypt import storage
+
+from letsencrypt.cli import choose_configurator_plugins, _renewal_conf_files, should_renew
+from letsencrypt.display import util as display_util, ops as display_ops
+from letsencrypt.plugins import disco as plugins_disco
+
+import traceback
+import logging.handlers
+import time
+from acme import jose
+import OpenSSL
```

Comment:
nit: Plugin ordering. Order is:
```
import builtins 

import dependencies

import our own code
```
Each section is alphabetized. Doing this is consistent across the project.